packages/perseus/src/validation.types.ts

```diff
@@ -51,10 +51,12 @@ export type PerseusDropdownUserInput = {
 export type PerseusExpressionRubric = {
     answerForms: ReadonlyArray<PerseusExpressionAnswerForm>;
     functions: ReadonlyArray<string>;
-};
+} & PerseusExpressionValidationData;
 
 export type PerseusExpressionUserInput = string;
 
+export type PerseusExpressionValidationData = Empty;
```

Comment:
I've been trying to remove things that are just aliases for `Empty`. I understand the impulse to have it though, so I'll ultimately defer to you (I just might forget and delete it in the future).